### PR TITLE
Use parquet as the default file format for iceberg

### PR DESCRIPTION
**Pull request by raunaqmorarka**
## Description
Use parquet as the default file format for iceberg

## Additional context and related issues
Parquet writer has comparable performance to ORC.
[Hive insert benchmark ORC vs parquet.pdf](https://github.com/trinodb/trino/files/11980809/Hive.insert.benchmark.ORC.vs.parquet.pdf)
Screenshot 2023-07-07 at 4 00 45 PM

Parquet reader has comparable or better performance to ORC.
[Iceberg unpartitioned sf1k orc vs parquet.pdf](https://github.com/trinodb/trino/files/11980825/Iceberg.unpartitioned.sf1k.orc.vs.parquet.pdf)
Screenshot 2023-07-07 at 4 01 36 PM

[Iceberg partitioned sf1k orc vs parquet.pdf](https://github.com/trinodb/trino/files/11980816/Iceberg.partitioned.sf1k.orc.vs.parquet.pdf)
Screenshot 2023-07-07 at 4 02 05 PM

Since there are no longer major performance benefits to staying on ORC and parquet is the popular format,
we can switch the default to parquet

## Release notes

( ) This is not user-visible or docs only and no release notes are required.
( ) Release notes are required, please propose a release note for me.
(x) Release notes are required, with the following suggested text:

```markdown
# Iceberg
* The default file format is changed to PARQUET. The catalog configuration `iceberg.file-format` can be used to specify a different file format as the default. ({issue}`18170`)
```

### CODE REVIEW

File: plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorSmokeTest.java [hunk 1]
```diff
@@ -97,7 +97,7 @@ public void testShowCreateTable()
                         "   comment varchar\n" +
                         "\\)\n" +
                         "WITH \\(\n" +
-                        "   format = '" + format.name() + "',\n" +
```
Comment (findepi) [inline plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorSmokeTest.java:100]: why this change?

File: plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergStatistics.java [hunk 2]
```diff
@@ -50,6 +51,7 @@ protected QueryRunner createQueryRunner()
     {
         return IcebergQueryRunner.builder()
                 .setInitialTables(NATION)
+                .setIcebergProperties(ImmutableMap.of("iceberg.file-format", "ORC"))
```
Comment (findepi) [inline plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergStatistics.java:54]: Document why. We should prefer to test with defaults, so any non-default setup should be documented, unless it's obviosu.

**Review by findepi — APPROVED** [review]

File: plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorSmokeTest.java (hunk 1 above)
Comment (raunaqmorarka) [reply plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorSmokeTest.java]: I got confused by some implementations of BaseIcebergConnectorSmokeTest not setting `"iceberg.file-format", format.name()`. I've removed this change and fixed the tests to set `"iceberg.file-format", format.name()`

File: plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergStatistics.java (hunk 2 above)
Comment (raunaqmorarka) [reply plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergStatistics.java]: The assertions in this class were not written to expect non-null values for data size since that is not implemented for ORC. I've now updated the assertions with data sizes instead of this change.

**Comment by raunaqmorarka** [comment]
/test-with-secrets sha=6bbe583a7ab288a42a8504d45624eb646392f600

**Comment by raunaqmorarka** [comment]
/test-with-secrets sha=b5428e572e40f82ab660e3faa051ad155ff816e2
